### ISSUE TITLE
Training with the random initialized teacher still get a good result

### ISSUE THREAD

**Issue by twmht** [ZJCV/overhaul#3]
I tried to distill resnet18 to resnet18-half (the channels in each layer is havled), but I forgot to initialize the teacher.

the result is still promising, the top1 accuracy can reach to 72%. 

what is your opinion about this?

From my opinion I think most of the `dummy` work are done by `transform_student` convolution, so it does not affect the student network too much. but this also raises a question that how many works are done by the `transform_strudent` convolution to make the loss convergence.

**Comment by zjykzj** [comment]
> I tried to distill resnet18 to resnet18-half (the channels in each layer is havled), but I forgot to initialize the teacher.
> 
> the result is still promising, the top1 accuracy can reach to 72%.
> 
> what is your opinion about this?
> 
> From my opinion I think most of the `dummy` work are done by `transform_student` convolution, so it does not affect the student network too much. but this also raises a question that how many works are done by the `transform_strudent` convolution to make the loss convergence.

Which dataset was used for training ? 

If you just using this repos' model implementation, the ResNet is inited by torchvision, using the ImageNet pretrained.

**Comment by twmht** [comment]
Hi, I use CIFAR100, you did not use CIFAR100?

**Comment by zjykzj** [comment]
> Hi, I use CIFAR100, you did not use CIFAR100?

I did, see README.md
![Screenshot from 2022-06-15 14-10-37](https://user-images.githubusercontent.com/69463217/173754953-73f10bce-2452-40ad-b696-785ca011cdac.png)

**Comment by twmht** [comment]
ok.
